### PR TITLE
Update Kestrel configuration sections

### LINKED ISSUE #10343: Update Kestrel configuration section
See: https://github.com/aspnet/AspNetCore/issues/6602#issuecomment-453633690

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 46a10a5c-bacc-1bcf-a7ae-499715e1bfdd
* Version Independent ID: 1585163b-b1d7-eeb8-ebc6-adfc2730f393
* Content: [Migrate from ASP.NET Core 2.1 to 2.2](https://docs.microsoft.com/en-us/aspnet/core/migration/21-to-22?view=aspnetcore-2.2&tabs=visual-studio)
* Content Source: [aspnetcore/migration/21-to-22.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/migration/21-to-22.md)
* Product: **aspnet-core**
* GitHub Login: @scottaddie
* Microsoft Alias: **scaddie**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #10343

Including a code sample should help, so I add one to the migration and Kestrel topics. Let me know if you would like to add more detail. This doesn't get into the weeds on **_why_**, which might be ok.

### CODE REVIEW

File: aspnetcore/migration/21-to-22.md [hunk 1]
```diff
@@ -107,6 +107,26 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
         });
 ```
 
+If the app doesn't call `CreateDefaultBuilder`, call <xref:Microsoft.AspNetCore.Hosting.WebHostBuilderKestrelExtensions.UseKestrel*> **before** calling `ConfigureKestrel`:
```
Comment (jkotalik) [inline aspnetcore/migration/21-to-22.md]: I don't think you need a code sample. I think just saying:

If the app doesn't call `CreateDefaultBuilder`, UseKestrel <xref:Microsoft.AspNetCore.Hosting.WebHostBuilderKestrelExtensions.UseKestrel*> still needs to be called, and should be **before** calling `ConfigureKestrel`.

Comment (guardrex) [reply aspnetcore/migration/21-to-22.md]: > don't think

I ~picture~ *code sample* is worth a thousand words. :smile:

I'll strike it, of course, if you like. There's no harm tho.

Comment (guardrex) [reply aspnetcore/migration/21-to-22.md]: The current text says what you just said without colloquial language ("still") and avoids the state verb ("be").

Comment (jkotalik) [reply aspnetcore/migration/21-to-22.md]: Haha okay keep it 😄 .

**Pull request merged**